server/belga/search_providers.py

```diff
@@ -303,7 +303,9 @@ def _get_newscomponent(self, item, component):
             return ''
 
     def _get_body_html(self, item):
-        return self._get_newscomponent(item, 'body')
+        # SDBELGA-393
+        body = '&nbsp;&nbsp;&nbsp;&nbsp;' + get_text(self._get_newscomponent(item, 'body'))
```

Comment:
why do we need 4 non-breaking spaces?

Reply:
It is to [add indentation for readability](https://user-images.githubusercontent.com/45254285/82974428-2dfba000-a004-11ea-853c-8294b9798f77.png), we think 4 space looks better than 2 space